Aire/View/NibClass/SceneView.swift

```diff
@@ -223,4 +263,31 @@ class SceneView: NibView {
 			}
 		}
 	}
+    
+    // camera position for adding labels
+    func makeUpdateCameraPos(towards: SCNVector3, pov: SCNNode) {
+        
+        // for any node in scene view
+        sceneView.scene.rootNode.enumerateChildNodes({ (node, _) in
```

Comment:
did you test this on very contaminated places? this may be too much computing and may affect performance gravely.

Reply:
I've not seen performance changes, testing in different environments, but we do need to monitor this more. We do need to enumerate all nodes, since we need to search for all of them, but added a stop command to stop processing after finding the node in the frame! Good catch